.gitmodules

```diff
@@ -0,0 +1,3 @@
+[submodule "website/themes/docsy"]
+	path = website/themes/docsy
+	url = https://github.com/google/docsy.git
```

Comment:
When you say consume it as a package do you mean uploading the files directly in the repo? We absolutely can do that, although this keeps our repo cleaner in my opinion. Open to other solutions :)

Reply:
My vote would probably be to containerize things, and pull docsy in as a dependency when building the dockerfile. But I'm not sure if there was a reason to use appengine standard

Reply:
Given that we own cloud-ops-sandbox.dev domain now, we can host the website wherever we want. So we can move it to firebase, keep it on AppEngine Standard, move to Flex - whatever makes sense for hosting static website and handles SSL for us "for free".

Reply:
Just talked with Daniel, and I would like to +1 to containerizing. Unless you think otherwise that's probably out of scope for this PR but is something that should be considered for the near future. Lmk your thoughts